Contribute/dotnet/labels-projects.md

```diff
@@ -108,18 +109,27 @@ Issues tagged for a specific release are noted with the `:checkered_flag: Releas
 
 Priority labels are all `P` followed by a single digit. Lower numbers are higher priority:
 
-- P0
-- P1
-- P2
-- P3
+- P0 - Indicates issues or PRs that are critical priority
+- P1 - High priority
+- P2 - Medium priority
+- P3 - Low priority
 
-### What about the other labels?
+### What about the other labels
 
 There are many other labels used by the content teams to manage different classifications of issues. If you're not on the content team, you can ignore these other labels.
 
 ## Projects
 
+Projects are intended for planning purposes, where prioritized work is automated through a Kanban board. Projects should only ever contain GitHub issues, _not_ pull requests. Projects differ from milestones, in that milestones only contain pull requests.
+
 We use projects in two ways:
 
-- Month YYYY project types: These are scrum boards for each month's working plan.
+- `Month YYYY` project types: These are Kanban boards for each month's working plan.
+  - Examples, [July 2020](https://github.com/dotnet/docs/projects/103), [August 2020](https://github.com/dotnet/docs/projects/117), and so on.
 - Long-running epics: These are used to organize tasks toward a goal when the work will occur over several months.
+  - Examples: [.NET 5 Wave - Reorganization](https://github.com/dotnet/docs/projects/105), [.NET Languages (.NET 5 wave)
+](https://github.com/dotnet/docs/projects/106), and so on.
+
+## Milestones
```

Comment:
As noted earlier, I'd omit this or explain how they're used.

Reply:
I do actually state how we use them.

Reply:
Where?